code/game/machinery/scp_294.dm

```diff
@@ -22,8 +22,14 @@
 	machine_flags = FIXED2WORK | EMAGGABLE | WRENCHMOVE
 	mech_flags = MECH_SCAN_FAIL
 
+	hack_abilities = list(
```

Comment:
already inherited from chem dispenser

Reply:
The chemistry dispenser doesn't have the emag ability.

Reply:
quite fine then